Color code US map

nmartinez23 opened the pull request:
- Added US map
- Added cursor pointer for map
- Added method to show politicians by clicking on a state
- Added method to change state's color if the majority of state's politicians are voting for CRA

holmesworcester commented:
My only other issue with this is that right now the map doesn't have enough explanation around it, because the legend ("red = team cable / green = team internet") doesn't really explain the deal with the states. 

Ideally the map would say "these are the states where we most need your help!" and a paragraph below would say "These 35 states have one undecided senator who could be our winning vote." or something like that. 

It could be in its own section and control its own view once people click on a state.

nmartinez23 commented:
@holmesworcester Yes it definitely makes sense. I remember Zeke mentioned something earlier about this so I'll meet with him tomorrow morning to see what the best course of action is. Thanks!

nmartinez23 commented:
@holmesworcester I added a media query to hide the map on smartphones and tablets in portrait mode. I also added the logic for the undecided states however I ran into a problem trying to add asterisks to the svg path for undecided states. 

There are two different undecided methods that are both commented out for now. The one that works lists out undecided states above the map. The other is the one I was trying to get asterisks added to the map. The "textPath" svg attribute will add an asterisk to an undecided state but will get overwritten by the next undecided state in the loop so only the very last undecided state in the loop will end up with an asterisk on the map. I'll keep looking to see if there's a better way.

nmartinez23 commented:
@holmesworcester just checking to see what your thoughts are on the most recent changes.
